### ISSUE TITLE
SyntaxError: Unexpected token '{' React SSR when importing external stylesheet in component

### ISSUE THREAD

**Issue by virkse** [webpack-contrib/css-loader#1066]
> I'm facing this issue, Can anyone help me please.

**Imported stylesheet as:**
import './Header.css';

**My webpack.config.js**
`const path = require("path");

const config = {
    mode: 'development',
    entry: {
        vendor: ["@babel/polyfill", "react"], // Third party libraries
        index: ["./src/components/entrypoints/index.js"]
        /// Every pages entry point should be mentioned here
    },
    output: {
        path: path.resolve(__dirname, "src", "public"), //destination for bundled output is under ./src/public
        filename: "[name].js" // names of the bundled file will be name of the entry files (mentioned above)
    },
    module: {
        rules: [
            {
                test: /\.(js|jsx)$/,
                use: {
                    loader: "babel-loader", // asks bundler to use babel loader to transpile es2015 code
                    options: {
                        presets: ["@babel/preset-env", "@babel/preset-react"] 
                    }
                },
                exclude: [/node_modules/, /public/]
            },
            {
                test: /\.(css|scss)$/,
                use: [ 'style-loader', 'css-loader' ]
            },
            {
                test: /\.jpe?g$|\.gif$|\.png$|\.svg$/i,
                loader: 'url-loader?limit=10000'
            },
        ]
    },
    resolve: {
        extensions: [".js", ".jsx", ".json", ".wasm", ".mjs", "*"]
    } // If multiple files share the same name but have different extensions, webpack will resolve the one with the extension listed first in the array and skip the rest.
};

module.exports = config;`

D:\learning\virkse-react-ssr\src\components\common\Header.css:2
[1] header{
[1]           ^
[1]
[1] SyntaxError: Unexpected token '{'
[1]     at Module._compile (internal/modules/cjs/loader.js:892:18)
[1]     at Module._extensions..js (internal/modules/cjs/loader.js:973:10)
[1]     at Object.newLoader [as .js] (D:\learning\virkse-react-ssr\node_modules\pirates\lib\index.js:104:7)
[1]     at Module.load (internal/modules/cjs/loader.js:812:32)
[1]     at Function.Module._load (internal/modules/cjs/loader.js:724:14)
[1]     at Module.require (internal/modules/cjs/loader.js:849:19)
[1]     at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (D:\learning\virkse-react-ssr\src\components\common\/Header.js:3:1)
[1]     at Module._compile (internal/modules/cjs/loader.js:956:30)
[1]     at Module._compile (D:\learning\virkse-react-ssr\node_modules\pirates\lib\index.js:99:24)
[1] [nodemon] app crashed - waiting for file changes before starting...

**Comment by evilebottnawi** [comment]
It looks like you just deleted our lovely crafted issue template. It was there for good reasons. Please help us solving your issue by answering the questions asked in this template. I'm closing this. Please either update the issue with the template and reopen, or open a new issue.

Problem in `pirates` package, we don't use that package, you don't provide full reproducible test repo so I can help, sorry

**Comment by virkse** [comment]
@evilebottnawi  i never used pirates anywhere. Here is my package.json file 

```
"dependencies": {
    "compression": "^1.7.4",
    "ejs": "^3.0.2",
    "express": "^4.17.1",
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "react-router-dom": "^5.1.2",
    "webpack": "^4.42.1"
  },
  "devDependencies": {
    "@babel/cli": "^7.8.4",
    "@babel/core": "^7.9.0",
    "@babel/node": "^7.8.7",
    "@babel/plugin-proposal-class-properties": "^7.8.3",
    "@babel/polyfill": "^7.8.7",
    "@babel/preset-env": "^7.9.0",
    "@babel/preset-react": "^7.9.4",
    "babel-loader": "^8.1.0",
    "concurrently": "^5.1.0",
    "css-loader": "^3.4.2",
    "mini-css-extract-plugin": "^0.9.0",
    "nodemon": "^2.0.2",
    "style-loader": "^1.1.3",
    "webpack-cli": "^3.3.11"
  }
```

**Comment by virkse** [comment]
@evilebottnawi  can you please help out me, if i added css loader perfectly then it should handle the css. Not sure what i should do.
